src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/CallHandlers/UnaryServerCallHandler.cs

```diff
@@ -44,6 +44,6 @@ protected override async Task HandleCallAsyncCore(HttpContext httpContext, JsonT
 
         serverCallContext.EnsureResponseHeaders();
 
-        await JsonRequestHelpers.SendMessage(serverCallContext, SerializerOptions, response);
+        await JsonRequestHelpers.SendMessage(serverCallContext, SerializerOptions, response, CancellationToken.None);
```

Comment:
httpContext.RequestAborted?

Reply:
All the async operations are on the HttpContext and friends. That cancellation token is implicitly used.